flycheck.el

```diff
@@ -3329,6 +3341,7 @@ current syntax check."
     (flycheck-stop))
   (flycheck-delete-all-overlays)
   (flycheck-clear-errors)
+  (flycheck-clear-displayed-error-messages)
```

Comment:
Isn't this something that `flycheck-clear-errors` should normally be doing? I find it weird to have to clear the errors and the displayed error messages separately.

Reply:
Same reason as https://github.com/flycheck/flycheck/pull/1972#issuecomment-1216166601. Clearing the errors data structures and displayed messages have to be distinct in order to prevent flickering.